Dockerfile

```diff
@@ -12,30 +12,53 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL C
 
-# Install essential packages.
+# OpenJDK repository
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends software-properties-common \
+  && rm -rf /var/lib/apt/lists/* \
+  && add-apt-repository ppa:openjdk-r/ppa
+
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
     build-essential \
-    man-db \
     pkg-config \
     libffi-dev \
     libssl-dev \
     curl \
-    kdiff3 \
     git \
-    vim \
```

Comment:
i use vim for debugging inside of the image. Can I get vim with `apt-get install vim`?